src/App.js

```diff
@@ -0,0 +1,67 @@
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import { NavBar } from "./Components/NavBar";
+import { Home } from "./Pages/Home";
+import { VideoPlayer } from "./Pages/VideoPlayer";
+import { SideBar } from "./Components/SideBar";
+import { useData } from "./Context/DataProvider";
+import { useEffect } from "react";
+import axios from "axios";
+import { BottomBar } from "./Components/BottomBar";
+import { History } from "./Pages/History";
+import { Snackbar } from "./Components/SnackBar";
+import { WatchLater } from "./Pages/WatchLater";
+import { LikedVideos } from "./Pages/LikedVideos";
+import { PlayList } from "./Pages/PlayList";
+import Modal from "./Components/Modal";
+import { CreatePlayList } from "./Components/CreatePlayList";
+export default function App() {
+  const {
+    state: { snackbarText },
+    openModal,
+    handleClose,
+    dispatch
+  } = useData();
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axios.get("/api/videos");
+        localStorage.setItem("data", JSON.stringify(response.data.videos));
+        const getData = JSON.parse(localStorage?.getItem("data"));
+        if (response.status === 200) {
+          dispatch({ type: "FETCH_DATA_TO_VIDEOS", payload: getData });
```

Comment:
Please rename this action type to FETCH_VIDEO_DATA or something else.

Reply:
I have changed it to FETCH_VIDEOS... is it more sensible now?

Reply:
Yup!